subhub/cfg.py

```diff
@@ -391,6 +391,13 @@ def ALLOWED_ORIGIN_SYSTEMS(self):
         """
         return self("ALLOWED_ORIGIN_SYSTEMS", "fake_origin1, fake_origin2").split(",")
 
+    @property
+    def PAYMENT_EVENT_LIST(self):
+        """"
+        PAYMENT_EVENT_LIST
+        """
+        return self("PAYMENT_EVENT_LIST", "test.system, test.event").split(",")
```

Comment:
Are these reasonable default values?

Reply:
This shows the pattern from Stripe, I thought that was sufficient.